modules/ztp-removing-content-from-managed-clusters.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-removing-content-from-managed-clusters_{context}"]
+= Changing applied managed cluster CRs using policies
+
+You can remove content from a custom resource (CR) deployed to a managed cluster through a policy.
+
+By default, all `Policy` CRs created from a `PolicyGenTemplate` CR have a `complianceType` set to `musthave`.
+In this mode, when you remove a field from a CR, the value is removed from the policy but is not removed from the CR on the managed cluster.
+A `musthave` policy without the removed field is still compliant because the CR on the managed cluster has all the specified content.
+
+To remove the content from the managed cluster, set the `complianceType` field to `mustonlyhave`.
```

Comment:
This reads like a step that should belong in the procedure section. Perhaps rephrasing the sentence to the following would help steer a docs user to the procedure, where the more granular steps exist?

> You can remove the content from the managed cluster by setting the `complianceType` field to `mustonlyhave`.

Reply:
Maybe I am reading this too closely, removing content from _the managed cluster_ and _from a custom resource_ the same?